Algorithm.CSharp/OpeningBreakoutAlgorithm.cs

```diff
@@ -288,7 +288,7 @@ private void ScanForEntrance()
 
             var allowedDollarLoss = MaximumPorfolioRiskPercentPerPosition * Portfolio.TotalPortfolioValue;
 
-            var shares = (int) (allowedDollarLoss/expectedCaptureRange);
+            var shares = allowedDollarLoss/expectedCaptureRange;
```

Comment:
Have balanced consistency and fractional support using generics in 7915b57. [This may not be the best approach](http://stackoverflow.com/a/8171547). I could instead add methods CalculateFractionalOrderQuantity() and SetFractionalHolding(). Any thoughts?